client/src/components/Pagination.jsx

```diff
@@ -31,11 +32,11 @@ const Pagination = ({ numReviews, paginate }) => {
   };
 
   return (
-    <div className="pagination-container">
+    <div className={styles.wrapper}>
       <ul>
         {pages.map((page) => (
-          <li key={page} className="pagination">
-            <input type="button" className={`page-button-${page} page-button`} onClick={(e) => onClickHandler(e)} value={page} />
+          <li key={page} className={styles.pagination}>
+            <input type="button" className={`pageButton-${page} ${styles.pageButton}`} onClick={(e) => onClickHandler(e)} value={page} />
```

Comment:
Does having the pageButton classname separate from CSS Modules cause any errors in style after Webpack runs?

I've never tried having one style use `styles` and another not, inside the same class name.

Reply:
I was concerned about this as well, but webpack seems to handle it fine. I had to handle it this way because the class name is dynamically set within the React component file. Definitely not ideal, but I'll take what works at this point :/

Reply:
Try template literals with bracket notation for the page variable if you want to also integrate pageButton to use styles :)

Reply:
Thanks for the suggestion, LeiLei! Will try that out.